### PR TITLE
New article: GPS Question types

### PR DESCRIPTION
**Pull request by exprin**
@joshuaberetta, please review this article

### CODE REVIEW

File: source/gps_questions.md [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+| :-------- | :--------------- | :--------------------------------- | :----------------- |
+| geotrace  | track            | Capture the GPS of the track       |                    |
+| calculate | distance         |                                    | distance(${track}) |
+| note      | display_distance | The distance is ${distance} Meters |                    |
+| survey    |
+
+### Calculating area from “Area” questions
+
+You can calculate the area from an “Area” question type using the "Calculate"
+question type and the `area()` function in the formbuilder as shown below:
```
Comment (joshuaberetta) [inline source/gps_questions.md:98]: Rather something like:
> You can calculate an area using the "Calculate" question type and the [`area()`](https://docs.getodk.org/form-operators-functions/#area) method, as shown below:

File: source/gps_questions.md [hunk 2]
```diff
@@ -0,0 +1,134 @@
...
+
+![Calculate distance](images/gps_questions/calculate_area.png)
+
+<section class="note">
+
+In the example above, the question “Capture the GPS of the boundaries” has been
```
Comment (joshuaberetta) [inline source/gps_questions.md:104]: Please update “Capture the GPS of the boundaries” as discussed above

File: source/gps_questions.md [hunk 3]
```diff
@@ -0,0 +1,134 @@
...
+added as an “Area” question type. The “Data Column Name” in the question
+settings has been changed to “boundary”.
```
Comment (joshuaberetta) [inline source/gps_questions.md:106]: You don't need to say "changed" as you are setting up the form from scratch and this implies that it was set to something else before and now you are making it something else. Rather use a word like "set" or just "the value is X".

File: source/gps_questions.md [hunk 4]
```diff
@@ -0,0 +1,134 @@
...
+
+The question <code>area(${boundary})</code> has been added as a “Calculate”
+question type. Its Data Column Name has been changed to “area”. The area from
+the calculation will be in square meters.
```
Comment (joshuaberetta) [inline source/gps_questions.md:110]: Please refer to my comment earlier about the `distance()` method.

File: source/gps_questions.md [hunk 5]
```diff
@@ -0,0 +1,134 @@
...
+
+The “Note” question is optional and has been added with the purpose of
+displaying the calculated area on the form.
```
Comment (joshuaberetta) [inline source/gps_questions.md:113]: Refer to earlier comment on the same sentence.

File: source/gps_questions.md [hunk 6]
```diff
@@ -0,0 +1,134 @@
...
+
+</section>
+
+In XLSForm, you can do the same, as follows:
+
+| type      | name         | label                                 | calculation       |
+| :-------- | :----------- | :------------------------------------ | :---------------- |
+| geoshape  | boundary     | Capture the GPS of the boundaries     |
+| calculate | area         |                                       | area(${boundary}) |
+| note      | display_area | The distance is ${area} Square Meters |                   |
```
Comment (joshuaberetta) [inline source/gps_questions.md:123]: Rather "square meters"

File: source/gps_questions.md [hunk 7]
```diff
@@ -0,0 +1,134 @@
...
+| survey    |
+
+<p class="note">
+  Download an XLSForm with examples from this article
```
Comment (joshuaberetta) [inline source/gps_questions.md:127]: > You can download an XLSForm...

**Review by joshuaberetta — CHANGES_REQUESTED** [review]

File: source/gps_questions.md [hunk 8]
```diff
@@ -0,0 +1,134 @@
+# GPS Question types
+
+KoboToolbox allows you to collect GPS coordinates as part of your data
```
[the remark this answers is not in the document]
Comment (exprin) [reply source/gps_questions.md:3]: I have rephrased this.

File: source/gps_questions.md [hunk 9]
```diff
@@ -0,0 +1,134 @@
...
+collection form. There are 3 GPS question types you can use, namely “Point”,
+“Line” and “Area”.
+
+Use a “Point” question type when you want to record a single GPS coordinate.
+This is perfect for questions where you need to show the location of a single
+feature on the earth’s surface such as a house, or a borehole.
```
[the remark this answers is not in the document]
Comment (exprin) [reply source/gps_questions.md:9]: I have removed that part

**Comment by exprin** [comment]
@joshuaberetta I just added a commit to this. Please review

**Review by joshuaberetta — APPROVED** [review]
@exprin please review the edits I made before merging.

**Pull request merged**